src/pip/_internal/req/req_file.py

```diff
@@ -23,7 +23,7 @@
 
 if MYPY_CHECK_RUNNING:
     from typing import (  # noqa: F401
-        Iterator, Tuple, Optional, List, Callable, Text
+        Iterator, Tuple, Optional, List, Callable, Text, NoReturn, Any
```

Comment:
FTR - I think it's worth it to contribute make isort handle this automatically.

None the less, I agree with @cjerdonek that these should be alphabetically ordered and doing them while modifying them is a good idea (since most of these are short lists).